source/apiVolontaria/apiVolontaria/local_settings_example.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import os
+
+from settings import *
+
+path = os.path.dirname(os.path.realpath(__file__))
+if path not in sys.path:
+    sys.path.append(path)
+
+DATABASES = {
```

Comment:
Maybe all these options can be put in a long comment to explain users how to use it.